src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -55,6 +55,42 @@ public function __construct($attributeName, $characterLimit)
     }
 }
 
+class ItemIdLengthException extends \RuntimeException
```

Comment:
All these newly added exceptions are the same except for a small difference in the exception message. Consider refactoring commonalities into a shared abstract parent, so the topic-specific exception classes can be as non-repetitive as possible.

Reply:
Moved all new exceptions into a seperate folder and inherited a parent exception. In the PHP Version update, the rest of the exception will be moved also.